tbot_contrib/swupdate/swupdate_script.py

```diff
@@ -0,0 +1,79 @@
+import asyncio
+import json
+import requests
+import websockets
```

Comment:
[CI is complaining about this import](https://travis-ci.org/github/Rahix/tbot/builds/664327942#L345-L346) because no typing information exists for it.  I think in this case, we can just suppress the warning with `# type: ignore`:

```suggestion
import websockets  # type: ignore
```